proto/viam/common/v1/common.proto

```diff
@@ -148,3 +150,10 @@ message WorldState {
 message ActuatorStatus {
   bool is_moving = 1;
 }
+
+extend google.protobuf.MethodOptions {
+  // safety_heartbeat_monitored is used on methods to signify that if a session is in use
+  // and the session was the last to call this method, the resource associated with the
+  // method will be stopped.
+  optional bool safety_heartbeat_monitored = 84260;
```

Comment:
is 84260 just a random number?

Reply:
it's viam on a phone pad plus a zero. @Otterverse was the inspiration